hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/io/nativeio/NativeIO.java

```diff
@@ -354,6 +347,19 @@ public boolean verifyCanMlock() {
       }
     }
 
+    public static void setConf(Configuration conf) {
+      workaroundNonThreadSafePasswdCalls = conf.getBoolean(
+          WORKAROUND_NON_THREADSAFE_CALLS_KEY,
+          WORKAROUND_NON_THREADSAFE_CALLS_DEFAULT);
+
+      cacheTimeout = conf.getLong(
+          CommonConfigurationKeys.HADOOP_SECURITY_UID_NAME_CACHE_TIMEOUT_KEY,
+          CommonConfigurationKeys.HADOOP_SECURITY_UID_NAME_CACHE_TIMEOUT_DEFAULT) *
+          1000;
+      LOG.debug("Initialized cache for IDs to User/Group mapping with a " +
```

Comment:
Use the logger format, even though we need to do the division by 1000. I would even change the message to show milliseconds and save the division.

Reply:
This is old code, not in scope of this change.